### ISSUE TITLE
Default email templates present even though not configured in kickstart

### ISSUE THREAD

**Issue by escii** [FusionAuth/fusionauth-issues#2485]
### Description
When using kickstart we expect to create email templates our self as mentioned in:
https://github.com/FusionAuth/fusionauth-issues/issues/532#issuecomment-602225908

Despite that, the latest kickstart is creating two email templates.

![image](https://github.com/FusionAuth/fusionauth-issues/assets/205567/30a76f57-85a1-4202-94d3-c9c4a6115939)

- Question 1: are these templates supposed to be created or is this a bug?
- Question 2: where can all required templates be found?

We assume https://fusionauth.io/docs/v1/tech/email-templates/templates-replacement-variables is the right location for all required email templates? 

When we checked the https://github.com/FusionAuth/fusionauth-example-kickstart we saw it does not contain all the templates.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

**Comment by Aaron-Ritter** [comment]
@mooreds we discovered that the documentation does not contain the latest templates provided when setting up FusonAuth without kickstart, Here a few examples:

```
diff --git a/terraform/email_templates/breached-password.html.ftl b/terraform/email_templates/breached-password.html.ftl

+[#setting url_escaping_charset="UTF-8"]

 -<a href="http://localhost:9011/password/forgot?email=${user.email}&tenantId=${user.tenantId}">
-  http://localhost:9011/password/forgot?email=${user.email}&tenantId=${user.tenantId}
+<a href="http://localhost:9011/password/forgot?client_id=${(application.oauthConfiguration.clientId)!''}&email=${user.email?url}&tenantId=${user.tenantId}">
+  http://localhost:9011/password/forgot?client_id=${(application.oauthConfiguration.clientId)!''}&email=${user.email?url}&tenantId=${user.tenantId}
 </a>

diff --git a/terraform/email_templates/breached-password.txt.ftl b/terraform/email_templates/breached-password.txt.ftl

+[#setting url_escaping_charset="UTF-8"]

-http://localhost:9011/password/forgot?email=${user.email}&tenantId=${user.tenantId}
+http://localhost:9011/password/forgot?client_id=${(application.oauthConfiguration.clientId)!''}&email=${user.email?url}&tenantId=${user.tenantId}

diff --git a/terraform/email_templates/change-password.html.ftl b/terraform/email_templates/change-password.html.ftl

-  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state' --]
-  [#assign url = "http://2id.online/password/change/${changePasswordId}?tenantId=${user.tenantId}" /]
-  [#list state!{} as key, value][#if key != "tenantId" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+       If we have an application context, append the client_id to ensure the correct application theme when applicable.
+  --]
+  [#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+  [#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]

diff --git a/terraform/email_templates/change-password.txt.ftl b/terraform/email_templates/change-password.txt.ftl

-[#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state' --]
-[#assign url = "http://2id.online/password/change/${changePasswordId}?tenantId=${user.tenantId}" /]
-[#list state!{} as key, value][#if key != "tenantId" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+       If we have an application context, append the client_id to ensure the correct application theme when applicable.
+  --]
+[#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]

diff --git a/terraform/email_templates/email-verification.html.ftl b/terraform/email_templates/email-verification.html.ftl

-  <a href="http://localhost:9011/email/verify/${verificationId}?tenantId=${user.tenantId}">
-    http://localhost:9011/email/verify/${verificationId}?tenantId=${user.tenantId}
+  <a href="http://localhost:9011/email/verify/${verificationId}?client_id=${(application.oauthConfiguration.clientId)!''}&postMethod=true&tenantId=${user.tenantId}">
+    http://localhost:9011/email/verify/${verificationId}?client_id=${(application.oauthConfiguration.clientId)!''}&postMethod=true&tenantId=${user.tenantId}

diff --git a/terraform/email_templates/email-verification.txt.ftl b/terraform/email_templates/email-verification.txt.ftl
 
-http://localhost:9011/email/verify/${verificationId}?tenantId=${user.tenantId}
+http://localhost:9011/email/verify/${verificationId}?client_id=${(application.oauthConfiguration.clientId)!''}&postMethod=true&tenantId=${user.tenantId}
```

**Comment by andrewpai** [comment]
@escii and @Aaron-Ritter, I'm unable to reproduce the email template creation issue on FusionAuth 1.55.1. I have a kickstart with no email templates defined, and after configuring FusionAuth with it, I see no email templates under `Customizations -> Email Templates.

Any chance you still have the kickstart file you were using when you filed this?

**Comment by Aaron-Ritter** [comment]
@andrewpai you are correct, the issue is solved apparently since fusionauth/1.53.x.

The last version i am able to reproduce it is with 1.52.1 https://github.com/FusionAuth/fusionauth-android-sdk/tree/fusionauth-1.52.1/fusionauth/1.52.1 and we can close this issue :tada:
